src/main/webapp/style.css

```diff
@@ -120,3 +120,64 @@ html * {
       margin-right: 0;
   }
 }
+
+#calendar{
```

Comment:
Not important but I personally would put a space between the name and the bracket.

Reply:
Replaced all over the file